src/server/constants.py

```diff
@@ -17,16 +17,27 @@ class PATHS:
     DATABASE = VAR_PATH + "database_ctb.db"
 
 
+@dataclass(frozen=True)
+class CONSTANTS:
+    """Variables which are constant through the lifetime of the app."""
+
+    DATABASE_NAME: str = os.getenv("CTB_DB_NAME", "")
+    DATABASE_USER: str = os.getenv("CTB_DB_USER", "")
+    DATABASE_PASSWORD: str = os.getenv("CTB_DB_PWD", "")
+    DATABASE_HOSTNAME: str = os.getenv("CTB_DB_HOST", "")
+    DATABASE_CONNECTION_TIMEOUT: int = int(os.getenv("CTB_DB_CONN_TMOUT", 30))
```

Comment:
If we're adding type hints to these constants, can you also add them to the rest of the file?